### PR TITLE
updated to SWIFT_VERSION 5.0

### PR DESCRIPTION
**Pull request by Radiksidenko**
updated pod to swift 5 for work with the new xcode

### CODE REVIEW

File: ParallaxView.podspec [hunk 1]
```diff
@@ -6,7 +6,7 @@ Pod::Spec.new do |s|
     s.summary = "Add parallax effect like in tvOS applications to any view."
     s.requires_arc = true
     s.license = "MIT"
-    s.version = "2.0.8"
+    s.version = "5.0.0"
```
Comment (RomanPodymov) [inline ParallaxView.podspec]: Why did you set `version` to 5.0.0? Probably you mix it up with [swift_version](https://guides.cocoapods.org/syntax/podspec.html#swift_versions).

Comment (Radiksidenko) [reply ParallaxView.podspec]: fixed it

File: ParallaxView.podspec [hunk 2]
```diff
@@ -6,7 +6,7 @@ Pod::Spec.new do |s|
     s.summary = "Add parallax effect like in tvOS applications to any view."
     s.requires_arc = true
     s.license = "MIT"
-    s.version = "2.0.8"
+    s.version = "5.0"
```
Comment (RomanPodymov) [inline ParallaxView.podspec]: Just leave `s.version` as it is and add `s.swift_version = '5.0'`.

Comment (tmm1) [reply ParallaxView.podspec]:
```suggestion
    s.version = "2.0.8"
```

Comment (Radiksidenko) [reply ParallaxView.podspec]: What makes you think that, sir?

Comment (tmm1) [reply ParallaxView.podspec]: That's what @RomanPodymov asked for above so this PR can be merged?

>Just leave s.version as it is

**Comment by tmm1** [comment]
Please update the README as well, since it still says Swift 3.0 is required.

**Pull request closed without merging**